ext/reflection/php_reflection.c

```diff
@@ -2112,20 +2112,22 @@ ZEND_METHOD(ReflectionGenerator, getExecutingGenerator)
 ZEND_METHOD(ReflectionParameter, __construct)
 {
 	parameter_reference *ref;
-	zval *reference, *parameter;
+	zval *reference;
+	zend_string *string_parameter = NULL;
+	zend_long position_parameter;
```

Comment:
I'd personally keep the name of this variable at `position` (just with different type now), don't think the `_parameter` really adds something. Similarly, `string_parameter` could be something like `arg_name`, to make it clearer what the parameter means in this case.